### ISSUE TITLE
Letters,writing during menus and ingame cause bugs/slowmess in the app

### ISSUE THREAD

**Issue by Jordanesquivez** [jpd002/Play-#509]
Firstly, all the games I've tried resulted in slowness due to text fonts and writing within the game ie; subtitles and menu options titles.
If a cutscene has 36-56 fps, when the subtitle appears it decreases to 5-10 fps.. also if the writing is within a image like a logo it runs normally.. I think something is happening with texts in the game it causing buggy and slowness

**Comment by jpd002** [comment]
Do you have an example of a game that behaves like that?

**Comment by Jordanesquivez** [comment]
yes these games.. naruto ultimate ninja 2.,odin sphere, grim grimoire,dragon Ball budokai tenkaichi... if the subtitle is in a box it will run smoothly. but if it's just the writng alone is normal

**Comment by Jordanesquivez** [comment]
yes these games.. naruto ultimate ninja 2.,odin sphere, grim grimoire,dragon Ball budokai tenkaichi... if the subtitle is in a box it will run smoothly. but if it's just the writng alone is normal
